src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/image.py

```diff
@@ -0,0 +1,50 @@
+from azure.cli.core.commands import LongRunningOperation
+from knack.util import CLIError
+from azure.mgmt.containerregistry.v2018_02_01_preview.models import (
+    ImportImageParameters,
+    ImportSource
+)
+from ._utils import (
+    validate_managed_registry,
+    get_resource_id_by_registry_name
+)
+
+IMPORT_NOT_SUPPORTED = "Image imports are only supported for managed registries."
+
+def acr_image_import(cmd,
+                     client,
+                     registry_name,
+                     image=None,
+                     resource_id=None,
+                     source_image=None,
+                     target_tags=None,
```

Comment:
`target_tag`

Reply:
Are you sure target_tag, not target_tags as the swagger names it?

Reply:
I can do multiple tags for the image import with the help of --t tag1 --t tag2